src/components/views/avatars/RoomAvatar.js

```diff
@@ -126,9 +126,16 @@ module.exports = React.createClass({
     },
 
     getFallbackAvatar: function(props) {
-        if (!this.props.room) return null;
```

Comment:
Interesting that this was using `this.props` here, when the props are being passed in explicitly. Other functions in `RoomAvatar (`getRoomAvatar` and `getOneToOneAvatar`) also do this and I wonder if they should also be changed.